### ISSUE TITLE
Readme file missing Requirements description

### ISSUE THREAD

**Issue by RagueI** [Automattic/simplenote-electron#3077]
### Expected

Expected all requirements are listed in the readme
### Observed

many errors when using `npm install` on Windows 10
### Reproduced

1.
2.
3.

### Where did you see the bug

- System Make: 
- System Model:
- OS: Windows10
- OS version: 21H1
- Browser (if applicable): 
- Browser version (if applicable): 
- Simplenote app version: 2.21.0
# log attached
[2022-02-23T21_04_40_467Z-debug-0.log](https://github.com/Automattic/simplenote-electron/files/8127837/2022-02-23T21_04_40_467Z-debug-0.log)

**Comment by whitevegagabriel** [comment]
+1

I had to install python27 and rename the EXE to python2.exe

Nonetheless, after that, I got hit with
```
npm ERR! gyp ERR! UNCAUGHT EXCEPTION
npm ERR! gyp ERR! stack Error: spawn C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\15.0\Bin\MSBuild.exe ENOENT
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
npm ERR! gyp ERR! stack     at onErrorNT (node:internal/child_process:476:16)
npm ERR! gyp ERR! stack     at processTicksAndRejections (node:internal/process/task_queues:83:21)
npm ERR! gyp ERR! System Windows_NT 10.0.22000
npm ERR! gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\***\\***\\***\\simplenote-electron\\node_modules\\node-sass\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
npm ERR! gyp ERR! cwd C:\***\***\***\simplenote-electron\node_modules\node-sass
npm ERR! gyp ERR! node -v v17.9.0
npm ERR! gyp ERR! node-gyp -v v3.8.0
npm ERR! gyp ERR! This is a bug in `node-gyp`.
npm ERR! gyp ERR! Try to update node-gyp and file an Issue if it does not help:
npm ERR! gyp ERR!     <https://github.com/nodejs/node-gyp/issues>
npm ERR! Build failed with error code: 7

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\***\AppData\Local\npm-cache\_logs\2022-05-20T15_11_18_830Z-debug-0.log
```

**Comment by allelimo** [comment]
I've got the same situation, starting with node-sass etc.
did you find any solution?

**Comment by allelimo** [comment]
installed ubuntu in virtualbox, same error after npm install:

npm ERR! code 1
npm ERR! path /home/osboxes/simplenote-electron/node_modules/node-sass
npm ERR! command failed
npm ERR! command sh -c node scripts/build.js

**Comment by whitevegagabriel** [comment]
I am not having any issues. I cannot seem to repro your issue.

I have followed the following steps:
- Installed latest VirtualBox
- Downloaded Ubuntu (
https://releases.ubuntu.com/22.04/ubuntu-22.04-desktop-amd64.iso)
- Installed Ubuntu on VirtualBox with 4GB Ram and 40 GB disk
- Ran these commands in Ubuntu:
sudo apt install nodejs
sudo apt install npm
sudo apt install git
git clone https://github.com/Automattic/simplenote-electron.git
cd simplenote-electron
npm install

What are some things that we might be doing differently?

Gabriel White-Vega

**Comment by passionate2023** [comment]
I'm also getting the same exception

**Comment by codebykat** [comment]
This should be fixed in #3183
